### PR TITLE
add logger to reconciler

### PR DESCRIPTION
**Pull request by avi-biton**

### CODE REVIEW

File: cmd/main.go [hunk 1]
```diff
@@ -136,6 +136,7 @@ func main() {
 	if err = (&controller.NotificationServiceReconciler{
 		Client:   mgr.GetClient(),
 		Scheme:   mgr.GetScheme(),
+		Log:      mgr.GetLogger(),
```
Comment (yftacherzog) [inline cmd/main.go:139]: The controller later on calls `r.Log.WithName()`. I think that creates a new logger instance. I wonder if that needs to change as well.

Comment (avi-biton) [reply cmd/main.go:139]: Thats the logger output:
```
2024-07-23T15:06:00Z    INFO    setup   starting manager
2024-07-23T15:06:00Z    INFO    starting server {"kind": "health probe", "addr": "[::]:8081"}
2024-07-23T15:06:00Z    INFO    Starting EventSource    {"controller": "pipelinerun", "controllerGroup": "tekton.dev", "controllerKind": "PipelineRun", "source": "kind source: *v1.PipelineRun"}
2024-07-23T15:06:00Z    INFO    Starting Controller     {"controller": "pipelinerun", "controllerGroup": "tekton.dev", "controllerKind": "PipelineRun"}
2024-07-23T15:06:00Z    INFO    Starting workers        {"controller": "pipelinerun", "controllerGroup": "tekton.dev", "controllerKind": "PipelineRun", "worker count": 1}
2024-07-23T15:06:00Z    INFO    Notification controller Reconciling PipelineRun {"Name": "test-component-on-push-j4j2z"}
2024-07-23T15:06:00Z    INFO    Removing finalizer	{"pipelineRun": "test-component-on-push-j4j2z"}
2024-07-23T15:06:00Z    INFO    Notification controller Reconciling PipelineRun {"Name": "test-component-on-push-qj2tn"}
2024-07-23T15:06:00Z    INFO    Removing finalizer	{"pipelineRun": "test-component-on-push-qj2tn"}
2024-07-23T15:06:00Z    INFO    Notification controller Reconciling PipelineRun {"Name": "test-component-on-push-s47pq"}
2024-07-23T15:06:00Z    INFO    Removing finalizer	{"pipelineRun": "test-component-on-push-s47pq"}
2024-07-23T15:06:00Z    INFO    Notification controller Reconciling PipelineRun {"Name": "test-component-on-push-v2fgm"}
2024-07-23T15:06:00Z    INFO    Removing finalizer	{"pipelineRun": "test-component-on-push-v2fgm"}
2024-07-23T15:06:00Z    INFO    Notification controller Reconciling PipelineRun {"Name": "test-component-on-push-qps4s"}
2024-07-23T15:06:00Z    INFO    Removing finalizer	{"pipelineRun": "test-component-on-push-qps4s"}
2024-07-23T15:06:00Z    INFO    Notification controller Reconciling PipelineRun {"Name": "test-component-on-push-8pxmk"}
2024-07-23T15:06:00Z    INFO    Removing finalizer	{"pipelineRun": "test-component-on-push-8pxmk"}
2024-07-23T15:06:13Z    INFO    Notification controller Reconciling PipelineRun {"Name": "test-component-on-push-qm7gj"}
2024-07-23T15:06:13Z    INFO    Adding finalizer        {"pipelineRun": "test-component-on-push-qm7gj"}
2024-07-23T15:06:14Z    INFO    Finalizer was added     {"pipelineRun": "test-component-on-push-qm7gj"}
2024-07-23T15:10:11Z    INFO    Notification controller Reconciling PipelineRun {"Name": "test-component-on-push-qm7gj"}
2024-07-23T15:10:12Z    INFO    Notifier        message was sent successfully   {"message": "[{\"name\":\"PIPELINERUN_NAME\",\"value\":\"test-component-on-push-qm7gj\"},{\"name\":\"IMAGE_URL\",\"value\":\"quay.io/avi_test/user-ns2/test-co
mponent/test-component:d39fb281670b814b6ba56d016f354c6b5fc36c55\"},{\"name\":\"IMAGE_DIGEST\",\"value\":\"sha256:6ddc56cb73552fb2182815f168d5b35123f901e0c87ea74348406a894e47d049\"},{\"name\":\"CHAINS-GIT_URL\",\"value\":\"https://github.c
om/avi-biton/testrepo\"},{\"name\":\"CHAINS-GIT_COMMIT\",\"value\":\"d39fb281670b814b6ba56d016f354c6b5fc36c55\"},{\"name\":\"JAVA_COMMUNITY_DEPENDENCIES\",\"value\":\"\"}]"}
2024-07-23T15:10:12Z    INFO    Notification controller SNS Notified
2024-07-23T15:10:12Z    INFO    Adding annotation	{"pipelineRun": "test-component-on-push-qm7gj"}
2024-07-23T15:10:12Z    INFO    Annotation was added    {"pipelineRun": "test-component-on-push-qm7gj"}
2024-07-23T15:10:12Z    INFO    Removing finalizer	{"pipelineRun": "test-component-on-push-qm7gj"}
2024-07-23T15:10:13Z    INFO    Finalizer was removed   {"pipelineRun": "test-component-on-push-qm7gj"}
2024-07-23T15:10:13Z    INFO    Notification controller Reconciling PipelineRun {"Name": "test-component-on-push-qm7gj"}
2024-07-23T15:10:13Z    INFO    Removing finalizer	{"pipelineRun": "test-component-on-push-qm7gj"}
2024-07-23T15:10:13Z    INFO    Finalizer was removed   {"pipelineRun": "test-component-on-push-qm7gj"}
```
It adds the name of the logger in front of the message

Comment (yftacherzog) [reply cmd/main.go:139]: alright

**Review by yftacherzog — APPROVED** [review]

**Comment by yftacherzog** [comment]
lgtm

**Pull request merged**